checks/taskcluster/write_secrets.py

```diff
@@ -0,0 +1,63 @@
+"""
+A check to verify that the Secrets service is operational.
+
+Information about the lastest indexed task is returned.
+"""
+import logging
+from datetime import timedelta
+
+import taskcluster
+import taskcluster.aio
+import taskcluster.exceptions
+
+from poucave import utils
+from poucave.typings import CheckResult
+
+from . import utils as tc_utils
+
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_NAME = "project/taskcluster/secrets-test"
```

Comment:
This could lead to a conflict if the health check url is called concurrently by e.g. two different users. This could be mitigated by giving the secret a unique name, e.g. if the secret name includes a slugid.